src/lightning-stubs/datatable/datatable.js

```diff
@@ -8,6 +8,7 @@ import { LightningElement, api } from 'lwc';
 
 export default class Datatable extends LightningElement {
     @api columns;
+    @api columnWidthsMode;
```

Comment:
Please move this up above `columns` to maintain alphabetical order.

Reply:
Done!